shell/edit/cloudcredential.vue

```diff
@@ -214,6 +217,25 @@ export default {
       this.credCustomComponentValidation = passed;
     },
 
+    getExtensions() {
```

Comment:
strictly speaking this should be `setExtensions` given it applies them directly to this.extensions and does not return them.

though ion functional programming it's better to keep it as `getExtensions` and return the collection, then assign to this.extensions in fetch